src/realtime/updater.py

```
@@ -411,11 +494,30 @@ def process_parsed_records_batch(self, records: list[Dict], timeseries: bool = F
+        owns_batch_transaction = not self.database.is_transaction_active()
         for table_name, rows in grouped.items():
+            if table_name in self.STRICT_RECORD_TABLES:
+                try:
+                    inserted += insert_wf_native_batch(
+                        self.database,
+                        table_name,
+                        rows,
+                        commit_batch=False,
+                        optimized=False,
+                    )
+                except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)